FirebaseSessions/SourcesObjC/NanoPB/FIRSESNanoPBHelpers.h

```diff
@@ -16,6 +16,8 @@
 #ifndef FIRSESNanoPBHelpers_h
 #define FIRSESNanoPBHelpers_h
 
+#import <Foundation/Foundation.h>
```

Comment:
Why are we adding this now? Is this something we added manually?

Reply:
Yeah I was surprised by this, but for this to build under SPM it needed this import. I was getting a failure that like NSObject doesn't exist before adding this. I believe Cocoapods may be implicitly including it.